metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -191,7 +180,7 @@ func createMLJob(host string) error {
 		return nil
 	}
 
-	req, err := http.NewRequest("PUT", "http://"+host+jobURL, strings.NewReader(mlJob))
+	req, err := http.NewRequest("PUT", "http://"+host+jobURL, bytes.NewReader(mlJob))
```

Comment:
This is more of question for my learning, not a review suggestion: why is `bytes.NewReader` more appropriate here than `strings.NewReader`?

Reply:
The main difference is that one takes a `[]byte` array and the other on a  `string` as param. ReadFile above returns a `[]byte` array and previously it was a string (mlJob var).